src/test/cypress/global.d.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference types="cypress" />
+interface TestHour {
+    calendarId: string;
+    startDate: string;
+    startTime: string;
+    endDay: string;
+    endTime: string;
+}
+
+interface TestException {
+    calendarId: string;
+    name: string;
+    startDate: string;
+    endDate: string;
+    openings: {
+        exceptionId: string,
+        startDate: string;
+        startTime: string;
+        endDay: string;
+        endTime: string;
+    }[]
+}
+
+interface TestCalendarReqBody{
+    name: string;
+    startDate: string;
+    endDate: string;
+    assignments: string[];
+    normalHours: TestHour[];
+    exceptions: TestException[];
+}
+
+
+declare namespace Cypress {
+    interface Chainable {
+        login(username: string, password: string): void;
+        loginAsAdmin(): void;
+        openCalendarSettings(isLoggedIn?: boolean): void;
+        createCalendar(reqBody: TestCalendarReqBody, callback: (res: Response<any>) => void): void;
```

Comment:
Can we use something other than `any`?  The response should be the same as the request, with an extra `id` returned.